api/proto/teleport/legacy/types/types.proto

```diff
@@ -2297,6 +2296,10 @@ message AuthPreferenceSpecV2 {
   // SecondFactors is a list of supported multi-factor types.
   // 1 is "otp", 2 is "webauthn", 3 is "sso",
   // If unspecified, the current default value is [1], or ["otp"].
+  //
+  // WARNING: only set SecondFactors if your cluster is fully upgraded to v17+.
```

Comment:
```suggestion
  // WARNING: only set second_factors if your cluster is fully upgraded to v17+.
```

Reply:
This ends up populating user facing documentation in the terraform docs tho.

Reply:
I see how that is awkward, but if anything that points to this type of docs generation being non-ideal. We would need to change line 2296 as well to get the right output for terraform docs:

> `second_factors` (List of Number) SecondFactors is a list of supported multi-factor types...